### PR TITLE
mcs: add config api for tso service

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Ref #5839.

### What is changed and how does it work?

This PR adds a config API so that TiDB can use query to get the config of the TSO server.

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: pkg/mcs/tso/server/apis/v1/api.go [hunk 1]
```diff
@@ -248,3 +255,13 @@ func GetKeyspaceGroupMembers(c *gin.Context) {
...
 	c.IndentedJSON(http.StatusOK, members)
 }
+
+// @Tags     config
+// @Summary  Get full config.
+// @Produce  json
+// @Success  200  {object}  server.Config
+// @Router   /config [get]
+func getConfig(c *gin.Context) {
+	svr := c.MustGet(multiservicesapi.ServiceContextKey).(*tsoserver.Service)
```
Comment (JmPotato) [inline pkg/mcs/tso/server/apis/v1/api.go:265]: Is it possible to panic here?

Comment (rleungx) [reply pkg/mcs/tso/server/apis/v1/api.go:265]: seem we need to use Get?

Comment (rleungx) [reply pkg/mcs/tso/server/apis/v1/api.go:265]: But I think it barely happens.

**Review by JmPotato — APPROVED** [review]

**Pull request merged**